Source/NSMenuView.m

```diff
@@ -1545,6 +1545,7 @@ - (BOOL) _trackWithEvent: (NSEvent*)event
   BOOL justAttachedNewSubmenu = NO;
   BOOL subMenusNeedRemoving = YES;
   BOOL shouldFinish = YES;
+  BOOL shouldStop = NO;
```

Comment:
Where does this value ever get used? You seem to only set it, but never read it.

Reply:
I have removed that value. It was an artifact from debugging apologies.